tests/gluon/test_gluon_model_export.py

```diff
@@ -213,6 +217,7 @@ def test_gluon_model_serving_and_scoring_as_pyfunc(gluon_model, model_data):
         model_uri=model_uri,
         data=pd.DataFrame(test_data.asnumpy()),
         content_type=pyfunc_scoring_server.CONTENT_TYPE_JSON_SPLIT_ORIENTED,
+        extra_args=["--no-conda"],
```

Comment:
A workaround for https://github.com/mlflow/mlflow/pull/3786#issuecomment-739682036

Reply:
This change also allows us to run this test against the dev verson of `mxnet`.